src/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -408,7 +408,9 @@ internal enum ExceptionArgument
         format,
         culture,
         comparer,
-        comparable
+        comparable,
+        source,
+        state
```

Comment:
This needs to be added to the switch statement in the other part of the file as well.